install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -2515,6 +2515,8 @@ operator:
   extraVolumeMounts: []
   # -- Annotations to be added to all top-level cilium-operator objects (resources under templates/cilium-operator)
   annotations: {}
+  # -- HostNetwork setting
```

Comment:
I believe you will have to run `make -C install/kubernetes/ && make -C Documentation update-helm-values` to update the docs for this new value

Reply:
Thank you, I updated the doc.